airflow/__init__.py

```diff
@@ -80,6 +80,11 @@ def __getattr__(name):
     manager.initialize_providers_hooks()
     manager.initialize_providers_extra_links()
 
+if settings.EXECUTE_LISTENERS_ON_SCHEDULER:
+    from airflow.plugins_manager import integrate_listener_plugins
+
+    integrate_listener_plugins()
```

Comment:
Given the name of this setting I wonder if this code should be somewhere in SchedulerJob instead?

Reply:
Actually, I've removed it for now. It will be easy to add it later.